tests/python/frontend/tensorflow/test_forward.py

```diff
@@ -63,24 +89,32 @@ def run_tvm_graph(graph_def, input_data, input_node, num_output=1,
                                                  layout=layout,
                                                  shape=shape_dict,
                                                  outputs=out_names)
-    with relay.build_config(opt_level=opt_level):
-        graph, lib, params = relay.build(mod, target, target_host, params)
-
-    ctx = tvm.context(target, 0)
-    from tvm.contrib import graph_runtime
-    m = graph_runtime.create(graph, lib, ctx)
-    # set inputs
-    for e, i in zip(input_node, input_data):
-        m.set_input(e, tvm.nd.array(i))
-
-    m.set_input(**params)
-    # execute
-    m.run()
-    # get outputs
-    assert out_names is None or num_output == len(out_names), (
-        "out_names: {} num_output: {}".format(out_names, num_output))
-    tvm_output_list = [m.get_output(i).asnumpy() for i in range(num_output)]
-    return tvm_output_list
+    if mode == 'interp':
+        ex = relay.create_executor("debug", mod=mod, ctx=tvm.cpu(), target="llvm")
+        inputs = []
+        for param in mod['main'].params:
+            inputs.append(tvm.nd.array(params[param.name_hint]))
+        result = ex.evaluate()(*inputs)
+        return vmobj_to_list(result)
+    else:
```

Comment:
could you please help add `mode =='vm'` here?

Reply:
done